README.md

```diff
@@ -3,4 +3,7 @@
 ## Goal
 Our goal is to randomly generate an algorithm to scramble a Rubik's Cube, example shown below.
 
-Scramble:  U  D2  B'  F'  D  F'  D  U'  B2  R'  B'  R  D2  F2  U'  L2  U  L  R'  D2  F2  D  B2  D2  L
\ No newline at end of file
+Scramble:  U  D2  B'  F'  D  F'  D  U'  B2  R'  B'  R  D2  F2  U'  L2  U  L  R'  D2  F2  D  B2  D2  L
```

Comment:
If there had been a newline at the end of this file, we wouldn't see this in the diff. Now when someone git blames this file, it will look like I wrote this line. Please always add newlines and the end of every file.

Reply:
I didn't realize that applied to markdown files like README.md(s) as well. Won't happen again.